app/src/test/java/me/hufman/androidautoidrive/TestSpotifyMusicAppController.kt

```diff
@@ -106,13 +107,6 @@ class TestSpotifyMusicAppController {
 
 	@Test
 	fun testCustomActions() {
-		controller.customAction(controller.CUSTOM_ACTION_TURN_REPEAT_ALL_ON)
```

Comment:
Would it make sense to leave these in place?

Reply:
It does, those should be re-added and I must have forgotten about that when I re-added the custom action repeat item. Thanks for catching that!